roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -43,13 +51,12 @@ def tags(self):
         """
         return []
 
-    @classmethod
-    def is_active(cls, task_vars):  # pylint: disable=unused-argument
+    def is_active(self):  # pylint: disable=unused-argument,no-self-use
```

Comment:
Have you tried keeping it as a `classmethod`? Ah, the class have no idea what is `task_vars` so it must not be a `classmethod`, I see.

It's unfortunate to fiddle with pylint, but we can probably just let it go here, since we have reasons to go against the rule (this is a base implementation meant to be overridden).